### DISCUSSION TITLE
the regex for svg tags doesn't work

### DISCUSSION THREAD

**Discussion by sambhav-aggarwal** [moalamri/vscode-inline-fold#97] [Q&A] [answered]
`{
  "inlineFold.regex":  "<svg(\\s*.*?\\s*)<\/svg>",
  "inlineFold.regexFlags": "gs",
  "inlineFold.regexGroup": 1
}`

This config doesn't work for svg tag folding. https://regex101.com/r/cG2hY4/1
What am I missing?

**Comment by moalamri** [comment, marked as answer]
~~I’m not sure if it is the reason, but have you tried to set regex group as a string “1” instead of a number?~~

I don’t think it’s the reason. I have checked your example link, it worked after I removed the escape character https://regex101.com/r/WBSjrQ/1

**Comment by sambhav-aggarwal** [comment]
Thank you! It is now working. 

However, it still doesn't fold the new line characters. The code folds but leave the new empty lines.
<img width="344" alt="Screenshot 2023-02-01 at 6 41 22 PM" src="https://user-images.githubusercontent.com/4591834/216052717-8b05e046-a4d5-42d2-960c-930faf1bac4a.png">

These are my current settings
<img width="530" alt="Screenshot 2023-02-01 at 6 43 44 PM" src="https://user-images.githubusercontent.com/4591834/216052943-b4d78d12-10d7-4f4e-9614-59034b3172e1.png">

**Reply by moalamri** [reply]
Great! Was it the escaping? 
Regarding your other question, please refer to https://github.com/moalamri/vscode-inline-fold/discussions/69#discussioncomment-4150206

**Reply by sambhav-aggarwal** [reply]
Yes, it was the escaping. I entered the expression in the settings UI which was double escaped in the JSON settings. Makes sense regarding the multiple line folding. Thank you, it's a great plugin.
